File: PLD1_InputProcessOutput/EX6_salesTax/salesTax.php
<?php
$stateTax = 0.04;
$countryTax = 0.02;

$purchase = readline("Amount Purchase : $");

$totalStateTax = $purchase * $stateTax;
$totalCountryTax = $purchase * $countryTax;
$totalSalesTax = $totalStateTax + $totalCountryTax;
$totalSale = $purchase + $totalSalesTax;

echo "Purchase          : $" . $purchase . "\n";
echo "State Sales Tax   : $" . $totalStateTax . "\n";
echo "Country Sales Tax : $" . $totalCountryTax . "\n";
echo "Total Sales Tax   : $" . $totalSalesTax . "\n";
echo "Total Sale        : $" . $totalSale . "\n";

File: PLD1_InputProcessOutput/EX6_salesTax/salesTax.cpp
#include <iostream>
using namespace std;

int main()
{
    float stateTax = 0.04;
    float countryTax = 0.02;
    float totalStateTax;
    float totalCountryTax;
    int purchase;
    int totalSalesTax;
    int totalSale;

    cout << "Amount Purchase : $";
    cin >> purchase;
    cout << endl;

    totalStateTax = purchase * stateTax;
    totalCountryTax = purchase * countryTax;
    totalSalesTax = totalStateTax + totalCountryTax;
    totalSale = purchase + totalSalesTax;

    cout << "Purchase          : $" << purchase << endl;
    cout << "State Sales Tax   : $" << totalStateTax << endl;
    cout << "Country Sales Tax : $" << totalCountryTax << endl;
    cout << "Total Sales Tax   : $" << totalSalesTax << endl;
    cout << "Total Sale        : $" << totalSale << endl;

    return 0;
}